Resources/doc/format_listener.rst

```diff
@@ -36,7 +36,7 @@ Setting ``priorities`` to a non-empty array enables Accept header negotiations.
                 # setting fallback_format to null means that in case of a priority mismatch the next rule will be considered
                 - { path: '^/admin', methods: [ 'GET', 'POST'], priorities: [ 'xml', 'html'], fallback_format: ~, prefer_extension: false }
                 # you can specifically target the exception controller
-                - { path: '^/api', priorities: [ 'xml', 'json'], fallback_format: xml, attributes: [ _controller: FOS\RestBundle\Controller\ExceptionController ], prefer_extension: false }
+                - { path: '^/api', priorities: [ 'xml', 'json'], fallback_format: xml, attributes: { _controller: FOS\RestBundle\Controller\ExceptionController }, prefer_extension: false }
```

Comment:
Any idea why do the original examples add a white space only after the opening bracket? `[ 'xml', 'json']` Why not `[ 'xml', 'json' ]` or `['xml', 'json']`?